javascript/javascript1/week1/index.js

```diff
@@ -0,0 +1,72 @@
+//Task1: Future age calculator 
+
+let yearOfBirth = 1988;
+// used let to check with other yearofbirth and other futureyear values. 
+let yearFuture = 2030;
+
+const age = yearFuture-yearOfBirth;
+/*
+console.log('\"You will be '+ age + ' years old in ' + yearFuture+ '.\"')
+*/
+const stringText = '\"You will be '+ age + ' years old in ' + yearFuture+ '.\"';
+console.log(stringText)
+
+//TASK2: A dog age calculator
+
+const dogYearOfBirth = 2015; //fixed. No assignment of new values.
+let dogYearFuture = 2025; //can assign and check with other years.
+let dogYear = dogYearFuture-dogYearOfBirth;
+
+let shouldShowResultInDogYears;
+if(shouldShowResultInDogYears == true){
+    dogYear *= 7;
+    console.log(`\"Your dog will be ${dogYear} dog years in ${dogYearFuture}.\"`)   
+} else {
+    console.log(`\"your dog will be ${dogYear} human years old in ${dogYearFuture}.\"`)
+}
+
+//Task3:House price estimator.
+
+let widthOfHouse;
+widthOfHouse = [8,5];
+let heightOfHouse;
+heightOfHouse = [10,8];
+let depthOfHouse;
+depthOfHouse = [10,11];
+let gardenSizeInM2;
+gardenSizeInM2 = [100,70];
+
+//Peters house price
+let volumeInMeters1 = widthOfHouse[0]*heightOfHouse[0]*depthOfHouse[0];
+const amtPayingByPeter = 2500000;
+let priceOfPetersHouse;
+priceOfPetersHouse = volumeInMeters1 * 2.5 * 1000 + gardenSizeInM2[0] * 300;
+
+if (priceOfPetersHouse <= amtPayingByPeter){
+    console.log(`Actual price of the house is ${priceOfPetersHouse}.\nPeter is paying (${amtPayingByPeter}) too high for the house.`)
+    
+} 
```

Comment:
If Peter is paying the right amount for the house, will the logged message make sense? What do you think?